Reword and move instant defeat option (#664)
Heegu-sama/Homm3BG: sections/combat.tex

\end{itemize}
Players are able to start multiple Combats during their Turn.

> \note{17}{

Reviewer: ```suggestion
\note{15}{
```

>   If your Town or Settlement is attacked by an enemy Hero and your Hero is not on that Field, you may immediately \textbf{pay 8} \svg{gold} to fight a defending Combat \textbf{using only your Units}.

Reviewer: ```suggestion
  If your Town or Settlement is attacked by an enemy Hero and your Hero is not on that Field, you may immediately \textbf{pay 8 \svg{gold} to defend with only your Units}.
```

>   You cannot use your Deck during that Combat as your Main Hero is not present.

Reviewer: ```suggestion
  You cannot use your Deck during this Combat, as your Main Hero is not present.
```

  Paying this Gold represents the cost of transporting the army there.

  \vspace*{1em}

>   When a \pagelink{Secondary}{Secondary Hero} is attacked, they \textbf{may choose} to be \pagelink{Endcombat}{instantly defeated} instead of fighting a Combat, because this helps preserving the Units.

Reviewer: ```suggestion
  When a \pagelink{Secondary}{Secondary Hero} is attacked, they \textbf{may choose} to be \pagelink{Endcombat}{instantly defeated} instead of engaging in Combat, which helps preserve the Units.
```
